packages/app/src/store/run-all-specs-store.ts

```diff
@@ -0,0 +1,96 @@
+import { RUN_ALL_SPECS_KEY } from '@packages/types/src'
+import { gql, useMutation, useQuery } from '@urql/vue'
+import { defineStore } from 'pinia'
+import { computed, ref } from 'vue'
+import { useRouter } from 'vue-router'
+import { RunAllSpecsDataDocument, RunAllSpecsDocument } from '../generated/graphql-test'
+import { getSeparator, SpecTreeNode, UseCollapsibleTreeNode } from '../specs/tree/useCollapsibleTree'
+
+type ResolvedConfig = { value: any, from: 'string', field: string }[]
+
+gql`
+query RunAllSpecsData {
+  currentProject {
+    id
+    config
+    currentTestingType
+  }
+}
+`
+
+gql`
+mutation RunAllSpecs ($specPath: String!, $runAllSpecs: [String!]!) {
+  setRunAllSpecs(runAllSpecs: $runAllSpecs)
+  launchOpenProject(specPath: $specPath) {
+    id
+  }
+}
+`
+
+const isRunMode = window.__CYPRESS_MODE__ === 'run' && window.top === window
+
+export const useRunAllSpecsStore = defineStore('runAllSpecs', () => {
```

Comment:
Should we be consistent and use the "options" style store that we use for our other stores? Or is there a limitation preventing us from using that style here?

Reply:
Only way to execute the gql query without having to call an explicit action. I didn't want to convert to using the store but I would have had to refactor the InlineSpecsList to get the communication to work. I'll think on this a bit more as I'm not too happy with the implementation.

Reply:
I'm still not entirely clear why this isn't possible using the options style store, but happy to move forward as is. I also feel this implementation isn't ideal, but I can't really nail down why - other than the fact we had to touch so many files to implement it.

Anyway, we can add this to our mental list of "things to improve" if/when we get a chance to attack some of the technical debt. It would be nice if the stores could be all written in a consistent style, though - what's the blocker on the options style store?